processor/src/trace/virtual_bus/circuit/prover.rs

```diff
@@ -93,11 +93,12 @@ impl<E: FieldElement> EvaluatedCircuit<E> {
     pub fn output_layer(&self) -> [E; 4] {
         let last_layer = self.layer_polys.last().expect("circuit has at least one layer");
 
+        // TODO: Just send the  poly ?
```

Comment:
Yes let's do that, it would be more descriptive than `[E; 4]`

Reply:
Changed